code/game/objects/machinery/doors/poddoor.dm

```diff
@@ -337,3 +337,47 @@ GLOBAL_LIST_INIT(faction_to_campaign_door_signal, list(
 
 /obj/machinery/door/poddoor/nt_lockdown/red
 	code_color = MISSION_CODE_RED
+
+// urban Shutters
+/obj/machinery/door/poddoor/urban
+	icon = 'icons/obj/structures/prop/urban/urbanshutters.dmi'
+	icon_state = "almayer_pdoor1"
+	base_icon_state = "almayer_pdoor"
+	openspeed = 4
+
+/obj/machinery/door/poddoor/urban/open_shutters/Initialize()
+	. = ..()
+	if(opacity)
+		set_opacity(0)
+
+/obj/machinery/door/poddoor/urban/open_shutters
+	name = "\improper Shutters"
```

Comment:
all this improper usage is incorrect. Just call it "shutters".

Reply:
Not to defend CM's strangeness here, but our base type for shutters is also called "\improper Shutters"

Reply:
yeah, probably unchanged from CM era lol